client/src/components/SignUp.style.js

```diff
@@ -0,0 +1,60 @@
+import styled from 'styled-components';
+
+export const Main = styled.main`
+  background-color: var(--main-color, #f9d52f);
```

Comment:
can probably remove this default color

Reply:
Did you mean to remove `#f9d52f` from `background-color`?